encryptor/queryDataEncryptor_test.go

```diff
@@ -393,7 +411,104 @@ schemas:
 		},
 		// 28. update with data as simple string for postgresql
 		{
-			Query:             `UPDATE "TableWithoutColumnSchema" as "t" set "other_column"='%s', "specified_client_id"='%s', "zone_id"='%s', "default_client_id"='%s'`,
+			Query:             `UPDATE "tablewithoutcolumnschema" as "t" set "other_column"='%s', "specified_client_id"='%s', "zone_id"='%s', "default_client_id"='%s'`,
+			QueryData:         []interface{}{simpleStringData, simpleStringData, simpleStringData, simpleStringData},
+			ExpectedQueryData: []interface{}{simpleStringData, encryptedValue, encryptedValue, encryptedValue},
+			Normalized:        true,
+			Changed:           true,
+			ExpectedIDS:       [][]byte{specifiedClientID, zoneID, defaultClientID},
+			DataCoder:         &PostgresqlDBDataCoder{},
+			dialect:           postgresql.NewPostgreSQLDialect(),
+		},
+
+		// 29. different case table identifiers, postgresql
+		// should match, lowercase config identifier == lowercase SQL identifier
+		{
+			Query:             `UPDATE lowercasetable set "other_column"='%s', "specified_client_id"='%s', "zone_id"='%s', "default_client_id"='%s'`,
```

Comment:
can you add test cases where we test same rules for column names?

Reply:
can we test in unit-tests different cases for columns too? we test it integration tests but will be great to test it in unit-tests too. because they are faster and as it looks for me, not so difficult to add it